OMCompiler/toJulia/lexer.jl

```diff
@@ -15,10 +15,11 @@ keywordWithAction = [
 string   = re"\"([^\"\\x5c]|(\\x5c['\"?\\x5cabfnrtv]))*\""
 ident    = re"[_A-Za-z][_A-Za-z0-9]*|'([^'\\x5c]|(\\x5c.))+'"
 int      = re"[0-9]+"
+bool     = re "(tru|fals)e"
```

Comment:
@sjoelund You are the regexp master Martin, I am not sure if I am correct here :p